ion-schema/src/isl/mod.rs

```diff
@@ -945,4 +1007,50 @@ mod isl_tests {
         write!(&mut buf, "{}", range.into()).unwrap();
         assert_eq!(expected, String::from_utf8(buf).unwrap());
     }
+
+    const SKIP_LIST: [&str; 5] = [
+        "ion-schema-schemas/json/json.isl", // the file contains `nan` which fails on equivalence for two schemas
+        "ion-schema-tests/ion_schema_1_0/nullable.isl", // Needs `nullable` annotation related fixes
+        "ion-schema-tests/ion_schema_1_0/schema/import/import_inline.isl", // related to order of types in the schema file
+        "ion-schema-tests/ion_schema_2_0/imports/tree/inline_import_a.isl", // related to order of types in the schema file
+        "ion-schema-tests/ion_schema_2_0/imports/tree/inline_import_c.isl", // related to order of types in the schema file
+    ];
+
+    fn is_skip_list_path(file_name: &str) -> bool {
+        SKIP_LIST
+            .iter()
+            .map(|p| p.replace('/', &MAIN_SEPARATOR.to_string()))
+            .any(|p| p == file_name)
+    }
+
+    #[test_resources("ion-schema-tests/**/*.isl")]
+    #[test_resources("ion-schema-schemas/**/*.isl")]
+    fn test_write_to_isl(file_name: &str) {
```

Comment:
I think I know what it's doing, but can you add a comment describing how this is testing the writer trait impl?